### PR TITLE
fix(*): fix cloudDB display

### PR DESCRIPTION
**Pull request by rfougier**

### CODE REVIEW

File: client/app/resources/i18n/hosting/Messages_fr_FR.xml [hunk 1]
```diff
@@ -78,6 +78,7 @@
    <translation id="hosting_dashboard_service_offer__90_PLAN" qtlid="196036">90plan</translation>
    <translation id="hosting_dashboard_service_offer_ASPFREE" qtlid="196049">aspfree</translation>
    <translation id="hosting_dashboard_service_offer_BUSINESS" qtlid="314110">business 2010</translation>
+   <translation id="hosting_dashboard_service_offer_CLOUDWEBX_1">cloud web</translation>
```
Comment (jleveugle) [inline client/app/resources/i18n/hosting/Messages_fr_FR.xml]: `Cloud web` ?

Comment (rfougier) [reply client/app/resources/i18n/hosting/Messages_fr_FR.xml]: So, what is the correct offer name in this case ?

Comment (jleveugle) [reply client/app/resources/i18n/hosting/Messages_fr_FR.xml]: `Cloud Web`

Comment (rfougier) [reply client/app/resources/i18n/hosting/Messages_fr_FR.xml]: So why all other offer names are in lower case ?

**Pull request merged**